### ISSUE TITLE
JSX. Does not detect scopes in tag name

### ISSUE THREAD

**Issue by psxcode** [Microsoft/TypeScript-TmLanguage#627]
**[TS and JS Grammar Extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.typescript-javascript-grammar) version:**  0.0.41 / latest

**Code**
Does not assign scope to point symbol
Does not detect 'comps' variable access
```ts
import comps from './mycomps'
const element = <comps.MyComp />
```

**Comment by sheetalkamat** [comment]
This is currently not supported by https://github.com/babel/babel-sublime which is used in the tag detection. Till there is enough traction on this issue, it will remain as is. Thanks.
